platform/app/services/temporal/activities/update_write_completion_activity.rb

```
@@ -0,0 +1,35 @@
+
+      timeouts(
+        start_to_close: 60,
+        heartbeat: 20,
+        schedule_to_close: 120
+      )
+
+      def execute(sync_run_id:, write_record_ids:)
+        ActiveRecord::Base.transaction do
+          # Update write records status
+          SyncWriteRecord.where(id: write_record_ids).update_all(
+            status: :written,
```

rubocop (1.69.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)